keadm/cmd/keadm/app/cmd/util/debinstaller.go

```diff
@@ -72,10 +73,7 @@ func (d *DebOS) IsK8SComponentInstalled(kubeConfig, master string) error {
 // Untar's in the specified location /etc/kubeedge/ and then copies
 // the binary to excecutables' path (eg: /usr/local/bin)
 func (d *DebOS) InstallKubeEdge(options types.InstallOptions) error {
-	arch, err := getSystemArch()
-	if err != nil {
-		return err
-	}
+	arch := runtime.GOARCH
```

Comment:
we can get arch in  `installKubeEdge` function and remove arch paramterer

Reply:
fixed ：）